src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,69 @@
+package codeu.chat.util;
+
+import java.io.*;
+
+public final class Tokenizer {
+   private StringBuilder token;
+   private String source;
+   private int at;
+
+   public Tokenizer(String source) {
+      this.source = source;
+      at = 0;
+      token = new StringBuilder();
+   }
+   public String next() throws IOException {
```

Comment:
Style nit: Please be consistent about leaving one blank line between methods (as opposed to zero lines or two).